core/src/com/unciv/logic/city/CityConstructions.kt

```diff
@@ -162,7 +162,8 @@ class CityConstructions {
     
     fun addFreeBuildings() {
         // "Provides a free [buildingName] [cityFilter]"
-        for (unique in cityInfo.getMatchingUniques(UniqueType.ProvidesFreeBuildings)) {
+        for (unique in cityInfo.getLocalMatchingUniques(UniqueType.ProvidesFreeBuildings)) {
```

Comment:
GetlocalMatchingUniques should run conditional check itself, otherwise this line will appear in loads of places

Reply:
I'll fix this in the next PR on conditionals, if you don't mind. To fix this properly a lot needs to be changed, and I'd rather fix that on its own